Classes/Public/Manager/Objects/OCTCall.h

```diff
@@ -11,33 +11,33 @@
 #import "OCTFriend.h"
 #import "OCTToxAVConstants.h"
 
-typedef NS_ENUM(NSUInteger, OCTCallStatus) {
+typedef NS_ENUM(NSInteger, OCTCallStatus) {
     OCTCallStatusInactive = 0,
     OCTCallStatusDialing,
     OCTCallStatusIncoming,
     OCTCallStatusInSession,
 };
 
-@interface OCTCall : NSObject
+@interface OCTCall : OCTObject
 
 /**
  * OCTChat related session with the call.
  **/
-@property (strong, nonatomic, readonly) OCTChat *chat;
+@property OCTChat *chat;
 
 /**
  * Call status
  **/
-@property (nonatomic, assign, readonly) OCTCallStatus status;
+@property OCTCallStatus status;
 
 /**
  * Call state of friend.
  **/
-@property (nonatomic, assign, readonly) OCTToxAVCallState state;
+@property OCTToxAVCallState state;
```

Comment:
Instead of `state` you can create nice methods
```
- (BOOL)sendingAudio;
- (BOOL)sendingVideo;
- (BOOL)receivingAudio;
- (BOOL)receivingVideo;
```

It seems that `OCTToxAVCallStateError` and `OCTToxAVCallStateFinished` are irrelevant to the end-user.